v3/src/components/case-table/collection-table.tsx

```diff
@@ -39,10 +43,13 @@ export const CollectionTable = () => {
   return (
     <div className="collection-table">
```

Comment:
Any suggestions for what to rename this now that we have a container "collection-table-and-title"?

Maybe "collection-box"?